heroku-24/setup.sh

```diff
@@ -161,15 +127,12 @@ packages=(
   postgresql-client-16
   rename
   rsync
-  shared-mime-info
   socat
-  tar
```

Comment:
I presume it's removed here because it's already in `essentials`. My opinion, `tar` is critical to our systems and should be listed. If canonical decided it shouldn't be in `essentials` in ubuntu:26.04, I'd still want it to be in `heroku-26`.

Reply:
Yeah I was on the fence whether to remove the explicit entry for it. My reasoning was that when adding a new stack, we'd diff the `installed-packages-*.txt` for the old and new stacks and see the omission.

However, I'm fine with adding it back too.

Ref `coreutils`, it includes packages that would never be omitted from the base image, so they would never not include it IMO.

Reply:
> Ref `coreutils`, it includes packages that would never be omitted from the base image

For example, `env` and `cat`:
https://packages.ubuntu.com/noble/amd64/coreutils/filelist

Reply:
> My reasoning was that when adding a new stack, we'd diff the installed-packages-*.txt for the old and new stacks and see the omission.

That was part of my reservation I think. Yes, I did diff the installed packages during `heroku-24` development, but there's nothing in our process that makes that a mandatory part of the process for new versions. This is a manual thing that must be remembered.